tools/objc/libtool.sh

```diff
@@ -34,6 +34,15 @@ fi
 
 WRAPPER="${MY_LOCATION}/xcrunwrapper.sh"
 
+# Ensure 0 timestamping for hermetic results.
+export ZERO_AR_DATE=1
+
+if $(/usr/bin/dirname "$0")/libtool_check_unique "$@"; then
```

Comment:
Sorry I missed this before--is it possible to refer to this relative to $MY_LOCATION?

Given that this script may be invoked in multiple contexts (according to the logic above), it might be prudent to refer to this in a canonical way.

Reply:
👍